ONSdigital/census-rm-case-processor
# New case event fields

# Motivation and Context

Additional Case Event dataa is required.

# What has changed
"event_payload", "event_channel", "event_source", "event_transaction_id" fields added to Event domain model.
Receipt processing handles message headers




# How to test?
Check out and build a local docker image from this branch and attached PRs where required.
Start services using docker-dev and run tests




# Links
[Trello](https://trello.com/c/VhVMCJ2M/813-additional-columns-for-case-events-table-8)
[Linked PR](https://github.com/ONSdigital/census-rm-case-api/pull/13)
[Linked PR](https://github.com/ONSdigital/census-rm-pubsub/pull/27)
[Linked PR](https://github.com/ONSdigital/census-rm-acceptance-tests/pull/84)




# Screenshots (if appropriate):
